ci/install_protobuf.sh

```diff
@@ -38,5 +38,5 @@ wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/p
 tar zxf protobuf-cpp-${CPP_PROTOBUF_VERSION}.tar.gz --no-same-owner
 cd protobuf-${CPP_PROTOBUF_VERSION}
 ./configure
-make -j ${nproc} && make install
+make && make install
```

Comment:
We may use `make -j || make && make install` to reduce the compiling time.

Reply:
Thanks @owent

This part changed a bit back and forth, not sure which commit you looked at.

Currently the code is `make -j ${nproc) && make install`

When using `make -j ${nproc}`, which was wrong because there is no `nproc` _environment variable_, `make -j` was executed alone ... this killed the github CI.

`make -j ${nproc)`, which is used in different places already, correctly invokes the nproc _utility_ to return the number of cores. This works fine in CI now.

Reply:
Thanks for the explanation.LGTM now.